mlflow/models/evaluation/base.py

```diff
@@ -1307,18 +1307,25 @@ def evaluate(
                         def fn(model_input):
                             return model.predict(model_input)
 
+                  - None: This indicates a static dataset will be used for evaluation instead of a
```

Comment:
https://github.com/liangz1/mlflow/pull/1#discussion_r1341724551

data should never be None. We can make `model` optional.